nvtabular/inference/triton/ensemble.py

```diff
@@ -55,16 +57,16 @@ def export_tensorflow_ensemble(
         The base name of the various triton models
     model_path:
         The root path to write out files to
-    label_columns:
-        Labels in the dataset (will be removed from the dataset)
```

Comment:
Removing the label_columns as a parameter means you have to update all the criteo, rossmann, and movielens notebooks to reflect this change. This includes not just removing the label_columns from the export_tensorflow_ensemble code, but also updating the workflow to add the 'Target' tag to the label column instead.

We might want to keep this parameter, add a deprecated warning if its set - and then if its not set grab from the schema like you are doing here?

Reply:
Indeed, notebook changes are required. Keeping this around for backwards compatibility seems reasonable to me though. Would you apply the same logic to `cats` and `conts`, or are you thinking specifically about labels here?

Reply:
We actually ended up _adding_ `cats` and `conts` to `export_tensorflow_ensemble` to make everything work consistently. 🤷🏻